docs/oid-info.md

```diff
@@ -180,26 +180,26 @@ that Sigstore operates.
+| server_url + repository         | server_url + project_path | ??       | ??        | Source Repository URI                   | Should include a fully qualified repository URL.                                                                                                                                       |
+| sha                | sha | ??       | build_commit | Source Repository Digest                | An immutable reference to a specific version of the source code. Should include the digest type followed by the digest, e.g. `sha1:abc123`.                                            |
 | ref                | ref     | ??       | build_branch | Source Repository Ref                   | The source ref that the build run was based upon. For example: refs/head/main.                                                                                                         |
 | repository_id         | project_id     | ??       | ??        | Source Repository Identifier     | Stable identifier for the owner of the source repository.                                                                                                                                      |
-| server_url + repository_owner         | namespace_path     | ??       | ??        | Source Repository Owner URI     | Fully qualified URL for the owner of the source repository.                                                                                                                                     |
+| server_url + repository_owner         | server_url + namespace_path | ??       | ??        | Source Repository Owner URI     | Fully qualified URL for the owner of the source repository.                                                                                                                                     |
 | repository_owner_id         | namespace_id     | ??       | ??        | Source Repository Owner Identifier     | Stable identifier for the owner of the source repository.                                                                                                                                   |
-| server_url + workflow_ref       | ??     | ??       | ??        | Build Config URI             | A reference to the initiating build instructions.                                                                                                                                      |
-| workflow_sha       | ??     | ??       | ??        | Build Config Digest          | An immutable reference to the specific version of the top-level build instructions. Should include the digest type followed by the digest, e.g. `sha1:abc123`.                         |
+| server_url + workflow_ref       |  pipeline_ref | ??       | ??        | Build Config URI             | A reference to the initiating build instructions.                                                                                                                                      |
```

Comment:
To confirm, this is fully qualified with a URL?

Reply:
Yes. See https://gitlab.com/gitlab-org/gitlab/-/issues/404722 for more details.